scripts/base/frameworks/broker/main.zeek

```diff
@@ -86,6 +86,24 @@ export {
 	## ZEEK_BROKER_MAX_THREADS environment variable overrides this setting.
 	const max_threads = 1 &redef;
 
+    ## Max number of items we buffer at most per peer. What action to take when
+	## the buffer reaches its maximum size is determined by
+	## `peer_overflow_policy`.
+	const peer_buffer_size = 2048 &redef;
+
+    ## Configures how Broker responds to peers that cannot keep up with the
+    ## incoming message rate. Available strategies:
+	## - disconnect: drop the connection to the unresponsive peer
+	## - drop_newest: replace the newest message in the buffer
+	## - drop_oldest: removed the olsted message from the buffer, then append
+    const peer_overflow_policy = "disconnect" &redef;
```

Comment:
I wonder whether we should provide some tooling to help people understand their current buffer use. We have some of this in telemetry, but it might help to have a policy script that periodically reports buffer occupancy to broker.log, or some such. (Needn't happen in this PR)

Reply:
That new buffer is per peer. We currently only have a metric for the entire chain. We could (should?) also measure how much we buffer per peer, but that metric would have to use the Broker endpoint ID as a label dimension to be meaningful - and I don't know whether that'd be useful to the user. If we wrap it into a script that "resolves" the endpoint IDs to node names maybe? That's probably a separate discussion, though. :)